tests/model_fields/test_imagefield.py

```diff
@@ -181,6 +181,23 @@ def test_pickle(self):
         loaded_p = pickle.loads(dump)
         self.assertEqual(p.mugshot, loaded_p.mugshot)
 
+    def test_image_field_io_closed_file(self):
+        """
+        Opening 'image' property as Image object of a model's ImageField will fail
+        with an 'I/O operation on closed file' error. Regression for #13750
+        """
+        from .models import Profile
+        profile = Profile()
+        profile.image = self.file1
+
+        profile.save()
+
+        self.assertTrue(profile.image.closed)
+
+        profile.image.width
```

Comment:
Can you make this work adding `self.assertTrue(profile.image.closed)` after this line? I wouldn't expect accessing the image dimensions to leave the file opened.